go/sqltypes/value.go

```diff
@@ -284,6 +284,36 @@ func (v Value) EncodeSQL(b BinWriter) {
 	}
 }
 
+// EncodeSQLStringBuilder is identical to EncodeSQL but it takes a strings.Builder
+// as its writer, so it can be inlined for performance.
+func (v Value) EncodeSQLStringBuilder(b *strings.Builder) {
```

Comment:
We have two sets of duplicated code in this file that are crying out for generics.
I don't suppose it is worth generating them (since it is only two copies).